Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -188,6 +191,8 @@ public enum FeatureFlag: String, CaseIterable {
             false
         case .playerIsReadyToPlay:
             true
+        case .useMimetypePackage:
+            false
```

Comment:
Do you plan to just activate using the remote flag later on?

Reply:
Yes but I might enable this now